samples/nrf9160/nrf_cloud_mqtt_multi_service/child_image/mcuboot/boards/nrf9160dk_nrf9160_0_14_0.overlay

```diff
@@ -16,3 +16,7 @@
 		nordic,drive-mode = <NRF_DRIVE_H0H1>;
 	};
 };
+
+&mx25r64 {
```

Comment:
this can be removed, already added https://github.com/nrfconnect/sdk-nrf/pull/11241/commits/797224d57a517aca70355728398890fc36e8b536